tezos/context/src/working_tree/storage.rs

```diff
@@ -430,14 +435,16 @@ impl PointerToInode {
                     .with_hash_id(hash_id.map(|h| h.as_u64()).unwrap_or(0))
                     .with_is_commited(true)
                     .with_inode_id(inode_id.0)
-                    .with_offset(offset.map(|o| o.as_u64()).unwrap_or(0)), //.with_offset(0),
+                    .with_is_offset_available(offset.is_some())
+                    .with_offset(offset.map(|o| o.as_u64()).unwrap_or(0)),
```

Comment:
@sebastiencs I see here for example that 0 is used when the offset is not available, cannot that work as a sentinel value?